docs/user-interaction-config-file-spec.rst

```diff
@@ -0,0 +1,204 @@
+Specification of the user interaction configuration file
+========================================================
+
+:Version:
+    1.0
+
+:Authors:
+    Martin Kolman <mkolman@redhat.com>
+
+This specification aims to establish a configuration file format, that can be used
+to communicate information about which installation screens have been seen by the user
+during the installation. Optionally the configuration file might also contain information
+about which configuration options have been changed by the user and if the user has
+explicitly requested post installation setup tools to be disabled.
+
+While this configuration file is primarily meant to be read (and potentially changed)
+by *post* installation tools (such as for example Initial Setup and Gnome Initial setup),
+Anaconda will take an existing configuration file into account at startup. This is meant
+to accommodate tools that are run *before* Anaconda is started, such as a system-wide
+language selection tool that runs before Anaconda is started and which sets language
+for both Anaconda and a Live installation environment.
+
+
+Configuration file location
+---------------------------
+
+The user interaction configuration file is stored in: ``/etc/sysconfig/anaconda``
+
+
+General configuration file systax
+---------------------------------
+
+The configuration file is based on the INI file de-facto standard,
+eq.: key=value assignments and square bracket framed section headers.
+
+For Python programs this file format can be parsed and written by the ConfigParser[0] module availabe
+from the standrad library. Comparable INI file parsing and writing modules are available for
+most other programming languages.
+
+Example:
+
+::
+
+    top_level_key=some_value
```

Comment:
If you add some toplevel group then it would be much easier for GNOME to read and write this file, as it would allow us to use our standard keyfile parser, [GKeyFile](https://developer.gnome.org/glib/stable/glib-Key-value-file-parser.html), which is for a format extremely similar to ini, but requires that all keys and values be in some group. It would probably speed adoption if we could use GKeyFile.